src/components/layout/Header.js

```diff
@@ -67,6 +71,24 @@ function Header() {
                   {t(APP_NAME)}
                 </Typography>
               </Link>
+              <Link href={ALL_COLLECTION_ROUTE} className={classes.link}>
+                <Typography
+                  variant="h6"
+                  color="inherit"
+                  className={classes.title}
+                >
+                  {t('All Collections')}
+                </Typography>
+              </Link>
+              <Link href={MY_LIST_ROUTE} className={classes.link}>
+                <Typography
+                  variant="h6"
+                  color="inherit"
+                  className={classes.title}
+                >
+                  {t('My Lists')}
```

Comment:
Not sure about this naming. Would `My Collections` make sense? Or maybe you just include these list in the mainmenu directly? as a subsection. I would like to see the layout to have a better idea^^

Reply:
We could discuss the name together ; )